src/main/java/seedu/address/model/tag/Subject.java

```diff
@@ -109,6 +108,23 @@ public static boolean isValidSubjectName(String subjectName) {
         return subject != Subjects.INVALID;
     }
 
+    /**
+     * This method is used to make the subjectNames for the same subject to be consistent.
+     * @param subjectName user input subject name
+     * @return corrected subject name that follows our convention, return null for invalid user input subject name.
+     */
+    public static String getCorrectedSubjectName(String subjectName) {
```

Comment:
Good checking for correct subject name so that all subject names are consistent.